Student_BCOs/Yuan_Gao_BCO_000552.json

```diff
@@ -0,0 +1,227 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000552",
+    "checksum": "9a99b34b3993907382a17ccc92e609809be5649e484fdf42a597c2906d9093c8",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "RNA-Seq Analysis of Differential Splice Junction Usage and Intron Retentions by DEXSeq",
+        "version": "1.0",
+        "license": "http://creativecommons.org/licenses/by/4.0/",
+        "created": "2019-10-17T19:43:19.909943",
+        "modified": "2019-10-17T20:09:29.248821",
+        "contributors": [
+            {
+                "name": "Yuan Gao",
+                "orcid": "https://orcid.org/0000-0003-0564-3016",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "ygao61@gwu.edu"
+            }
+        ]
+    },
+    "usability_domain": [
+        "RNA-seq analysis of differential splice junction usage and intron retentions by DEXSeq. ",
+        "Adapted from Li, Y., Rao, X., Mattox, W. W., Amos, C. I., & Liu, B. "
```

Comment:
Need to include publication title and DOI, not just author list. 
"This BCO was adapted from [paper name] by [author list] [doi/PMID]

See #136